vagrant-spk

```diff
@@ -640,22 +644,38 @@ def publish(args):
     finally:
         os.remove(temp_spk)
 
-def halt(args):
+def warn_deprecated(command_name):
+    ANSI_RED = "\x1b[31m"
```

Comment:
At some point we're going to have some difficulty porting this to Python 3. It's A-OK with me for now, though.